internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,426 @@
+package asyncapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/antihax/optional"
+	"github.com/go-yaml/yaml"
+	"github.com/iancoleman/strcase"
+	"github.com/spf13/cobra"
+	spec2 "github.com/swaggest/go-asyncapi/spec-2.4.0"
+	yaml3 "k8s.io/apimachinery/pkg/util/yaml"
+
+	kafkarestv3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+
+	"github.com/confluentinc/cli/internal/pkg/ccloudv2"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/types"
+)
+
+const parseErrorMessage string = "topic is already present and overwrite flag is false"
+
+type flagsImport struct {
+	overwrite               bool
+	kafkaApiKey             string
+	schemaRegistryApiKey    string
+	schemaRegistryApiSecret string
+}
+
+type kafkaBinding struct {
+	XConfigs map[string]string `yaml:"x-configs"`
+}
+
+type Message struct {
+	SchemaFormat string      `yaml:"schemaFormat"`
+	ContentType  string      `yaml:"contentType"`
+	Payload      any         `yaml:"payload"`
+	Name         string      `yaml:"name"`
+	Tags         []spec2.Tag `yaml:"tags"`
+}
+
+type Spec struct {
+	Asyncapi   string           `yaml:"asyncapi"`
+	Info       spec2.Info       `yaml:"info"`
+	Servers    spec2.Server     `yaml:"servers"`
+	Channels   map[string]Topic `yaml:"channels"`
+	Components Components       `yaml:"components"`
+}
+
+type Components struct {
+	Messages        map[string]Message `yaml:"messages"`
+	SecuritySchemes interface{}        `yaml:"securitySchemes"`
+}
+
+type Security struct {
+	ConfluentBroker         []any `yaml:"confluentBroker"`
+	ConfluentSchemaRegistry []any `yaml:"confluentSchemaRegistry"`
+}
+
+type MessageRef struct {
+	Ref string `yaml:"$ref"`
+}
+
+type Operation struct {
+	OperationID string                        `yaml:"operationId"`
+	TopicTags   []spec2.Tag                   `yaml:"tags"`
+	OpBindings  spec2.OperationBindingsObject `yaml:"bindings"`
+	Message     MessageRef                    `yaml:"message"`
+}
+
+type Topic struct {
+	Description           string        `yaml:"description"`
+	Publish               Operation     `yaml:"publish"`
+	Subscribe             Operation     `yaml:"subscribe"`
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	Bindings              TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka kafkaBinding `yaml:"kafka"`
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import <input-file-name>",
+		Short: "Adds topics, schemas and tags from the yaml file.",
```

Comment:
Not sure if this is the right terminology, but it wouldn't hurt to be more specific here.
```suggestion
		Short: "Add topics, schemas, and tags from the AsyncAPI YAML specification file.",
```

Reply:
Can you include where these topics are getting added to? (Make it clear that this is modifying Confluent Cloud resources)